src/swmmanywhere/geospatial_utilities.py

```diff
@@ -637,21 +646,31 @@ def flwdir_whitebox(fid: Path) -> np.array:
         dem = temp_path / "dem.tif"
         shutil.copy(fid, dem)
 
+        wbt_root = temp_path / "WBT"
+
+        if wbt_zip_path:
+            prepare_wbt(wbt_root, wbt_zip_path, refresh_download=False)
+        else:
+            prepare_wbt(wbt_root)
```

Comment:
By default, `zip_path=None, refresh_download=False`. So, instead of the if-block, you could just do:

```
prepare_wbt(wbt_root, zip_path=wbt_zip_path)
```